diff --git a/dist/fontkit.es.js b/dist/fontkit.es.js
index 2711b07413d1eb095487d4fddc7273f4afc1d82b..6169bfb7be72a6dff39d79c1f89670f8cbf37ca5 100644
--- a/dist/fontkit.es.js
+++ b/dist/fontkit.es.js
@@ -29595,6 +29595,7 @@ var GPOSProcessor = /*#__PURE__*/function (_OTProcessor) {
   };
 
   _proto.applyAnchor = function applyAnchor(markRecord, baseAnchor, baseGlyphIndex) {
+    if (!baseAnchor || !markRecord || !markRecord.markAnchor) return;
     var baseCoords = this.getAnchor(baseAnchor);
     var markCoords = this.getAnchor(markRecord.markAnchor);
     var basePos = this.positions[baseGlyphIndex];
diff --git a/dist/fontkit.umd.js b/dist/fontkit.umd.js
index c18af0e3cfce5893cc0012cc3a4e2b69c78d4c95..c73d1dab86a9990a455c7f824da167bec887d4c0 100644
--- a/dist/fontkit.umd.js
+++ b/dist/fontkit.umd.js
@@ -37745,6 +37745,7 @@
 	  };
 
 	  _proto.applyAnchor = function applyAnchor(markRecord, baseAnchor, baseGlyphIndex) {
+    if (!baseAnchor || !markRecord || !markRecord.markAnchor) return;
 	    var baseCoords = this.getAnchor(baseAnchor);
 	    var markCoords = this.getAnchor(markRecord.markAnchor);
 	    var basePos = this.positions[baseGlyphIndex];
